Tests/test_file_apng.py

```diff
@@ -263,12 +263,9 @@ def test_apng_chunk_errors():
     with Image.open("Tests/images/apng/chunk_no_actl.png") as im:
         assert not im.is_animated
 
-    def open():
+    with pytest.warns(UserWarning):
         with Image.open("Tests/images/apng/chunk_multi_actl.png") as im:
             im.load()
```

Comment:
```suggestion
        with Image.open("Tests/images/apng/chunk_multi_actl.png") as im:
            im.load()
        assert not im.is_animated
```
Is there any particular reason you removed the `assert`?

Reply:
Accidentally :)